html/pfappserver/lib/pfappserver/Form/Config/Radiusd/TLSProfile.pm

```diff
@@ -67,6 +78,10 @@ sub options_ocsp {
     return  map { { value => $_, label => $_ } } @{pf::ConfigStore::Radiusd::OCSPProfile->new->readAllIds};
 }
 
+sub options_tls_version {
+    return map { { value => $_, label => $_ } } qw(1.0 1.1 1.2 1.3);
```

Comment:
- Is it possible to have values defined in another place ? Below, I saw that we take values from ConfigStore.
- I read some things about TLS 1.3, are we sure it makes sense to enable it ?